From 5f4aba639af9bc5573539de3c53aa66e36cc6801 Mon Sep 17 00:00:00 2001
From: Patrick Delaunay <patrick.delaunay@foss.st.com>
Date: Fri, 18 Apr 2025 16:18:17 +0200
Subject: [PATCH 1676/1695] dts: stm32: fix GPU OPP description

The overdrive OPP at 900MHz and 0.9V is not sustainable for
10 year life activity @100% activity rate. So use the nominal
OPP at 800MHz and 0.8V by default and let the user select
the overdrive OPP when it is possible.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git d724f425c0bc839d3df27fb3af5f7fb29de0b745 dts: stm32: fix GPU OPP description
dc74bac3cbb271e524cc55a8319327b59fa65e9c dts: stm32: fix GPU OPP description for STM32MP23x]
Fixes: d8e84f2e09ec ("dts: stm32: add GPU OPP description")
Signed-off-by: Patrick Delaunay <patrick.delaunay@foss.st.com>
Change-Id: Ic7028c922c32b7c1960f7783a902a7affb9bb627
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/452728
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
---
 core/arch/arm/dts/stm32mp255.dtsi | 1 -
 1 file changed, 1 deletion(-)

diff --git a/core/arch/arm/dts/stm32mp255.dtsi b/core/arch/arm/dts/stm32mp255.dtsi
index 9212171d3..0423678ac 100644
--- a/core/arch/arm/dts/stm32mp255.dtsi
+++ b/core/arch/arm/dts/stm32mp255.dtsi
@@ -20,7 +20,6 @@
 			opp-hz = /bits/ 64 <900000000>;
 			opp-microvolt = <900000>;
 			opp-supported-hw = <0x2>;
-			st,opp-default;
 		};
 	};
 };
-- 
2.47.2

